### PR TITLE
Update datamarts-understand.md

### PR DESCRIPTION
**Pull request by salilkanade**
Added a section for "Understand what's in the default dataset" and updated verbiage from "auto-generated dataset" to "default dataset"

### CODE REVIEW

File: powerbi-docs/transform-model/datamarts/datamarts-understand.md [hunk 1]
```diff
@@ -12,34 +12,44 @@ LocalizationGroup: Data from files
 ---
 # Understand datamarts
 
-This article describes and explains important concepts about datamarts.
+## Understand Dataset (default)
```
Comment (ShannonLeavitt) [inline powerbi-docs/transform-model/datamarts/datamarts-understand.md:15]: Blocking issue: An intro paragraph must appear between the H1 and the first H2.

File: powerbi-docs/transform-model/datamarts/datamarts-understand.md [hunk 2]
```diff
@@ -12,34 +12,44 @@ LocalizationGroup: Data from files
...
+Finally, if you don't want to use the default dataset directly, you can connect to the datamart’s SQL endpoint. For more information, see [Create reports using datamarts](datamarts-create-reports.md).
+
+### Understand what's in the default dataset
+
+Currently, tables in the datamart are automatically added to the default dataset. Users can also manually select tables 
+or views from the datamart they want included in the model for more flexibility. Objects that are in the default dataset 
+will be created as a layout in the model view.
+
+The background sync that includes objects (tables and views) will wait for the downstream dataset to not be in use to 
+update the dataset, honoring bounded staleness. Users can always go and manually pick tables they want or no want in
```
Comment (ShannonLeavitt) [inline powerbi-docs/transform-model/datamarts/datamarts-understand.md:51]:
```suggestion
update the dataset, honoring bounded staleness. Users can always go and manually pick tables they want or not want in 
```
Should "no" be "not" instead?

**Comment by ShannonLeavitt** [comment]
@davidiseminger 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerbi-docs/transform-model/datamarts/datamarts-understand.md (hunk 2 above)
Comment (salilkanade) [reply powerbi-docs/transform-model/datamarts/datamarts-understand.md:51]: yes. thanks for catching this

File: powerbi-docs/transform-model/datamarts/datamarts-understand.md (hunk 1 above)
Comment (salilkanade) [reply powerbi-docs/transform-model/datamarts/datamarts-understand.md:15]: we can add this sentence back in.

**Comment by salilkanade** [comment]
Why is this not accepted? Are there any blockers still? @davidiseminger

**Comment by salilkanade** [comment]
Why is this not accepted? Are there any blockers still? @davidiseminger

**Comment by davidiseminger** [comment]
Um... I'm the author.
#sign-off

**Pull request merged**